README.md

```diff
@@ -9,10 +9,12 @@ Batch downloader for payslips from Hilan.
     - Login to Hilan as you normally would and view your latest payslip.
     - Find a request that looks like `PaySlip2022-01.pdf?Date=01/01/2022&UserId=12341231231`.
     - Go to `Request Headers` on the bottom right and copy the `Cookie` header.
-    - Save the value to a file, e.g. `hilan.cookie`.
+    - Save the value to a file, e.g. `hilan.cookie` under `bin` directory.
```

Comment:
it's relative to where your working directory run the file from, not sure that's correct

Reply:
as foolish steps for the dumb users who are just following instructions and part of your example 🤷  
```
1. make
2. steal cookie
3. run getlush (which is under bin)
```
never mind I can remove it.